tests/qam-minimal/install_update.pm

```diff
@@ -19,8 +19,6 @@ use testapi;
 sub run {
     select_console 'root-console';
 
-    pkcon_quit;
-
```

Comment:
because limitations of script_run/assert_script_run - it uses return value of last command..

in this case it always return 0 --> so always pass